pages/getting-started/index.js

```diff
@@ -57,7 +57,7 @@ function CodePreview() {
 
 function Page({url}) {
   return (
-    <Layout pathname={url ? url.pathname : ''}>
+    <Layout pathname={url ? url.pathname : ''} topNav={true}>
```

Comment:
This is what I'm referring to:

![img_6064](https://user-images.githubusercontent.com/737065/27765515-4ad7006a-5e81-11e7-9b2a-37958b45697c.PNG)
![img_6065](https://user-images.githubusercontent.com/737065/27765516-4af1d16a-5e81-11e7-912f-9ba2536f0f2f.PNG)